airflow/cli/commands/celery_command.py

```diff
@@ -46,7 +46,7 @@
 WORKER_PROCESS_NAME = "worker"
 
 warnings.warn(
-    "Use celery command from providers package, Use celery provider > 3.5.2", DeprecationWarning, stacklevel=2
+    "Use celery command from providers package, Use celery provider > 3.6.0", DeprecationWarning, stacklevel=2
```

Comment:
Are you sure about the version ? Should it be >= 3.6.0 ?

Reply:
I guess [this change](https://github.com/apache/airflow/pull/36794) not include into the 3.6.0 and will be included into the 3.6.1

Reply:
So maybe better >= 3.6.1 ?

Also this one needs rebasing @pavansharma36  (and conflict resolving)